src/celldega/nbhd/__init__.py

```diff
@@ -205,3 +213,301 @@ def alpha_shape_geojson(gdf_alpha, meta_cluster, inst_alpha):
     geojson_alpha['inst_alpha'] = inst_alpha
 
     return geojson_alpha
+
+def create_hextile(radius, path_landscape_files=None, img_height=100, img_width=100):
```

Comment:
I don't think you need to change it now, but just a heads-up that we may change this later so we calculate the hex bins in micron space, as last time I met @cornhundred , he wanted everything (ALPH. HEX, GRAD) in micron space and only transform it to pixel space right before front-end rendering. I will comment a code snippet I tested on your Jira page.